util.py

```diff
@@ -9,6 +9,15 @@
 import pytz
 
 
+def get_dataset_id(dataset_name, **kwargs):
+    datasets = api.list_datasets(**kwargs)
+    for dataset in datasets:
```

Comment:
nit: can this for loop be condensed into python [d for d in ...] pattern?

Reply:
It could, but I didn't touch this code.  I just reorganized the file.

Reply:
👍